HolyshiftService/Dto/ResponseDto.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text.Json.Serialization;
+
+namespace HolyShift.Dto;
+
+public class ResponseDto
+{
+    [JsonPropertyName("success")]
+    public bool Success { get; set; } = true;
+
+    [JsonPropertyName("error_code")]
+    public string? ErrorCode { get; set; }
+
+    [JsonPropertyName("message")]
+    public string? Message { get; set; }
+
+    public static ResponseDto Error(string errorCode, string message)
```

Comment:
We can make this a template function to make it possible to construct a child classes of responseDto easily

will look something like this:
```C#
T Error<T>(...)
  where T: ResponseDto
{
  return new T{...}
}
```